values/gitea/gitea.gotmpl

```diff
@@ -45,6 +45,7 @@ gitea:
   podAnnotations:
     policy.otomi.io/ignore.init: psp-allowed-users
     policy.otomi.io/ignore.init-ca-finish: psp-allowed-users
+    backup.velero.io/backup-volumes: init,data,temp
```

Comment:
does it mean sense to backup `temp` ?

Reply:
or perhaps this annotation can be removed since we backup everything from namespace

Reply:
Since defaultVolumesToRestic flag is set, it is not mandatory to specify annotations. Later we can improve on excluding certain volumes from backup and add Velero hooks, to ensure consistent backups